app/models/uphold_connection.rb

```diff
@@ -47,6 +47,25 @@ class UpholdAccountState
       where("updated_at < ?", UPHOLD_ACCESS_PARAMS_TIMEOUT.ago)
   }
 
+  def self.initial_funnel_by_country
+    ActiveRecord::Base.connection.execute("
+      select count(id), country
+      from uphold_connections
+      group by country
+      order by count(*) desc
+    ")
+  end
+
+  def self.kycd_by_country
+    ActiveRecord::Base.connection.execute("
+      select count(id), country
+      from uphold_connections
+      where is_member = true
+      group by country
+      order by count(*) desc
+    ")
+  end
```

Comment:
I think this would make more sense being executed in the job rather than on the model, let's move it into  Cache::Metrics::UpholdKycByCountryJob